pyscitt/pyscitt/key_vault_sign_client.py

```diff
@@ -0,0 +1,101 @@
+import base64
+import hashlib
+import json
+from azure.identity import DefaultAzureCredential
+from pyasn1.type.namedtype import NamedTypes, NamedType
+from pyasn1.type.univ import Integer, Sequence
+from pyasn1.codec.der.encoder import encode
+
+from azure.keyvault.certificates import (
+    CertificateClient,
+    KeyVaultCertificate
+)
+from azure.keyvault.keys import (
+    KeyClient
+    )
+from azure.keyvault.keys.crypto import (
+    CryptographyClient,
+    SignatureAlgorithm,
+)
+
+class _DERSignature(Sequence):
+    """Internal helper class for decoding AKV signature for CCF."""
+    componentType = NamedTypes(
+        NamedType("r", Integer()),
+        NamedType("s", Integer()),
+    )
+
+class KeyVaultSignClient():
+    """MemberIdentity implementation that uses Azure Key Vault."""
+
+    def __init__(self, akv_sign_configuration: str):
+        akv_sign_configuration_dict = json.loads(akv_sign_configuration)
+        self._vault_name = akv_sign_configuration_dict['keyVaultName']
+        self._vault_url = f"https://{self._vault_name}.vault.azure.net"
+        self._identity_certificate_name = akv_sign_configuration_dict['cetificateName']
+        self._identity_certificate_version = akv_sign_configuration_dict['cetificateVersion']
+        self._identity_private_curve_key_size = akv_sign_configuration_dict['privateCurveKeySize']
+    
+    @staticmethod
+    def _decode_certificate(cert: KeyVaultCertificate) -> str:
+        decoded = base64.b64encode(cert.cer).decode()  # type: ignore
+
+        styled_cert = "-----BEGIN CERTIFICATE-----\n"
+        for index in range(0, len(decoded), 64):
+            if index + 64 > len(decoded):
+                styled_cert += decoded[index:]
+            else:
+                styled_cert += decoded[index : index + 64]
+            styled_cert += "\n"
+        styled_cert += "-----END CERTIFICATE-----"
+        return styled_cert
```

Comment:
Let's keep this simple:

```suggestion
        cert_pem = f"-----BEGIN CERTIFICATE-----\n{decoded}\n-----END CERTIFICATE-----"
        return cert_pem
```